CoreUtilities/src/au/gov/asd/tac/constellation/utilities/keyboardshortcut/RecordKeyboardShortcut.java

```diff
@@ -0,0 +1,53 @@
+package au.gov.asd.tac.constellation.utilities.keyboardshortcut;
+
+import java.util.ArrayList;
+import java.util.Optional;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.input.KeyCombination.Modifier;
+import javafx.scene.input.KeyEvent;
+import javafx.scene.layout.StackPane;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+
+/**
+ *
+ * @author spatel308
+ */
+public class RecordKeyboardShortcut {
+    
+    public Optional<String> start(Stage primaryStage) {
+        var label = new Label();
```

Comment:
`final`

Reply:
can we also declare as a `Label` rather than `var`